src/OctoshiftCLI.Tests/Integration/GenerateArchiveTests.cs

```diff
@@ -0,0 +1,28 @@
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.Integration
+{
+    public class GenerateArchiveTests
+    {
+        #region Script generation and verification helpers
```

Comment:
Using `#region` is discouraged nowadays because ppl usually hide their smelly code in region blocks instead of paying attention to the code smell and try to address it 😄